src/ts/ConvivaAnalytics.ts

```diff
@@ -61,6 +64,7 @@ export class ConvivaAnalytics {
         + 'Please load the Conviva script (conviva-core-sdk.min.js) before Bitmovin\'s ConvivaAnalytics integration.');
       return; // Cancel initialization
     }
+    this.hasPlayingEvent = Boolean(player.EVENT.ON_PLAYING);
```

Comment:
Blocks usually group code that somehow belongs together. Does this state flag belong to the assertion above? Please insert a blank line and then add a comment about why this event state is tracked.